huey/consumer.py

```diff
@@ -724,15 +725,19 @@ def check_worker_health(self):
 
     def _set_signal_handlers(self):
         signal.signal(signal.SIGTERM, self._handle_stop_signal)
+        signal.signal(signal.SIGINT, self._handle_stop_signal)
         if hasattr(signal, 'SIGHUP'):
             signal.signal(signal.SIGHUP, self._handle_restart_signal)
 
     def _handle_stop_signal(self, sig_num, frame):
-        self._logger.info('Received SIGTERM')
+        sig = signal._int_to_enum(sig_num, signal.Signals)
```

Comment:
I'm guessing this is a 3.x-only API? It's not documented in the 2.x documentation and so is probably a non-starter. It might be fine just to write a new signal handler for SIGINT instead of overloading the existing TERM handler to do both.